ods_ci/tests/Resources/OCP.resource

```diff
@@ -64,7 +64,8 @@ Verify Notebook CR Is Running
 Is RHODS Self-Managed
     [Documentation]     Returns ${FALSE} if RHODS Managed (i.e., Cloud version) is installed
     ...                 in the cluster (e.g., usually on OSD, ROSA).
-    ...                 Returns ${TRUE} if RHODS Self-Managed is installed (e.g., usually on PSI or baremetal)
+    ...                 Returns ${TRUE} if RHODS Self-Managed is installed or if PRODUCT=ODH
+    IF  "${PRODUCT}" == "ODH"  RETURN     ${TRUE}
```

Comment:
For the final return value, we could just do
```
Run Keyword And Return    Evaluate    not ${is_managed}
```
see https://robotframework.org/code/?codeProject=N4IgdghgtgpiBcIDCBXAzgFwPZQAQGMsATOAGhBLXwCcBLABw1qzARBHIDNaAbGNBAG1Q3PgDlocRBn4YAdNSwAjLBg4hCYGVrYAqfbgBqEOhCV80ufboA6YAGTAAYgHkXAX1xfcAaxgBPAEYAXgASYAAVahQYT28-fwAmMOcIHjRYuwABYAAPfwAvOK8EwO9fAMTyhIBmOztrXAjZXCQIDMtrO2bMXEC7ctcAJXLw3OLcAEkxcpz8ooHy70mnUdzcWjBcUNcXXAgwIm3duTH3RaWlgBksAHMmrFaWNCw+cqh-AH1CKCgDo4uSwAolcAMpAwGXLw3e4RR5IZ6vGDvL6qAAWMGo3xwf0OkO8QLEABFAYSiSB3ABdcgkOgANxgRAACooAFYwfBqeCcNIZciKFQYQyYtDMViIACscgADHJAhSgA

Reply:
I've added it. Thanks for the suggestion!